dependencies.yaml

```diff
@@ -0,0 +1,7 @@
+# Third-party components versions
+# Renovate bot will use this file to track and update component versions
+
+# renovate: datasource=github-release depName=prometheus/prometheus
+prometheus: v2.49.1
+# renovate: datasource=github-release depName=grafana/grafana
+grafana: v7.5.16
```

Comment:
The latest version is v7.5.17, but I want to make it updating automatic by "Renovate" bot.

Reply:
How do we know the new version of components works well with TiDB?

Reply:
just auto update for patch version upgrading, the minor or major upgrade should passed by strict testing.

Reply:
Also the bot's PR should be approved by human.